packages/types/src/base.ts

```diff
@@ -15,14 +15,14 @@ export enum ChainTypes {
   Osmosis = 'osmosis'
 }
 
-const supportedChainIds = [
-  'eip155:1',
-  'bip122:000000000019d6689c085ae165831e93',
-  'cosmos:cosmoshub-4',
-  'cosmos:osmosis-1'
-] as const
+export enum SupportedChainIds {
```

Comment:
I don't think `supportedChainIds`/`SupportedChainIds` is necessarily bad for now, plus we now have them indexed with a respective enum key.